searx/plugins/generic_url_rewrite.py

```diff
@@ -0,0 +1,73 @@
+# See https://github.com/searx/searx/issues/2609
+# Author: <a.mathew@outlook.com>
+
+from searx import searx_dir
+from searx.plugins import logger
+from os.path import join as pathJoin
+from flask_babel import gettext
+
+
+name = gettext("Hits rewrite")
+description = gettext("Rewrite hits to custom endpoints")
+default_on = False
+preference_section = "general"
+
+configPath = pathJoin(searx_dir, "plugins/genericRewrites/config.yml")
```

Comment:
Perhaps it would be better to create a section in the main `settings.yml` instead of adding a new config file without documentation and the option to configure the path. What do you think?